docs/dietpi_tools.md

```diff
@@ -24,11 +24,11 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 
 ### Software overview
 
-=== "Software Optimised"
+=== "Browse Software"
 
-    - Begin by selecting **Software Optimized** in the main menu list and hit ++enter++.
+    - Begin by selecting **Browse Software** in the main menu list and hit ++enter++.
 
-    - Scroll through the list of available software - for more details check [DietPi Optimised Software](../software/).
+    - Scroll through the list of available software - for more details check [DietPi Optimised Software items](../software/).
```

Comment:
What about getting rid of that "optimised" which does not appear in the menu anymore? E.g.
```
for more details check [the full software list](../software/).
```

Reply:
Changed from "DietPi Optimized Software items" to "DietPi software list".